.gitignore

```diff
@@ -1,3 +1,4 @@
+dropin.cache
```

Comment:
it's the cache of the plugin system in twisted
https://twistedmatrix.com/documents/current/core/howto/plugin.html#plugin-caching